### PR TITLE
Add iris-ued to conda-forge

### PR DESCRIPTION
**Pull request by LaurentRDC**
Ultrafast electron diffraction data exploration.

[Repository](https://github.com/LaurentRDC/iris-ued)
[Documentation](https://iris-ued.readthedocs.io/en/master/)

### CODE REVIEW

**Comment by LaurentRDC** [comment]
This recipe will not build until scikit-ued is added to the conda-forge channel (#6015)

File: recipes/iris-ued/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - h5py >=2.6.0
+    - pyqt >=5
+    - scikit-image >=0.13
+    - scikit-ued >=0.5.9
+    - pyqtgraph >=0.10
+    - qdarkstyle >=2.5.4
+    - npstreams >=1.5.2
+    - cached_property >=1.3
+
+test:
```
Comment (synapticarbors) [inline recipes/iris-ued/meta.yaml]: Can you test the entry point:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

Comment (LaurentRDC) [reply recipes/iris-ued/meta.yaml]: The entry point launches a GUI, which would run forever without any human input to close it (as it is implemented right now). Any idea on how to get around this?

Entry point: `python -m iris` is equivalent to `python -c 'from iris.gui import run; run()'`

Comment (synapticarbors) [reply recipes/iris-ued/meta.yaml]: missed that this was a gui script. you can leave out the test.

File: recipes/iris-ued/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  name: 'iris-ued'
+  version: '{{ version }}'
+
+source:
+  fn: 'iris-ued-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/i/iris-ued/iris-ued-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  skip: True  # [not win]
```
Comment (mariusvniekerk) [inline recipes/iris-ued/meta.yaml]: you probably want to combine these selectors.  This area is technically a dictionary so you could be clobbering your own skip

**Pull request merged**